docs/plugin/install/help.go

```diff
@@ -0,0 +1,8 @@
+package install
+
+const Description = "Install plugin."
+
+var Usage = []string{"jfrog plugin install <plugin name>"}
+
+const Arguments string = `	plugin name
```

Comment:
Users should be abel to install a specific plugin version.
For example, to install version 1.0.0 of my-plugin:
```
jfrog plugin install my-plugin@1.0.0
```

Reply:
Added an explanation. Please tell me what you think